cli/command/image/save.go

```diff
@@ -47,8 +47,15 @@ func NewSaveCommand(dockerCli command.Cli) *cobra.Command {
 	return cmd
 }
 
-// RunSave performs a save against the engine based on the specified options
+// RunSave performs a save against the engine based on the specified options.
+//
+// Deprecated: this function was only used internally, and will be removed in the next release.
 func RunSave(ctx context.Context, dockerCli command.Cli, opts saveOptions) error {
```

Comment:
Oh! I think I forgot to push; this one doesn't need a deprecation first as it's already unusable due to the un-exported `saveOptions` type; let me check my local branch